autoload/denops/_internal/job.vim

```diff
@@ -58,24 +57,23 @@ else
           \ 'err_cb': funcref('s:out_cb', [a:options.on_stderr, 'stderr']),
           \ 'exit_cb': funcref('s:exit_cb', [a:options.on_exit, 'exit']),
           \}
-    return job_start(a:args, l:options)
+    let l:job = job_start(a:args, l:options)
+    if l:job->job_status() !=# "run"
+      throw printf("Invalid value for argument cmd: '%s' is not executable", a:args[0])
```

Comment:
What happen when `a:args` is like `['deno', 'unknowncommand']`? If it also fail, I think we should include entire `a:args` instead of `a:args[0]` for better debugging.

Reply:
The only time `job_status` will be other than 'run' immediately after `job_start` is when the executable file does not exist on Windows. If the executable file exists and the job terminates with an error, processing branches to `on_exit`.

Reply:
And this error message mimics the original Vim error message.

Reply:
But this line will be changed because of differences between Windows and Linux.

Reply:
Got it. Just ignore my comment 👍